src/model/validation.model.ts

```diff
@@ -33,6 +33,12 @@ export interface DateValidationType {
         DayYear: ValidationError,
         MonthYear: ValidationError,
         DayMonthYear: ValidationError,
+    },
+    RuleBased?: {
+        Underage: ValidationError,
```

Comment:
Should underage also be optional?

Reply:
There are underage rules for both date of birth and date of appointment